code/datums/keybinding/mob.dm

```diff
@@ -226,3 +240,21 @@
 	if(!user.mob) return
 	user.body_l_leg()
 	return TRUE
+
+/datum/keybinding/mob/prevent_movement
+	key = "Ctrl"
+	name = "block_movement"
+	full_name = "Block movement"
+	description = "Prevents you from moving"
```

Comment:
Needs clarification. The var isn't much better either. This seems like it *disables* movement period.

Reply:
that's exactly what it does
you have to rebind your face direction keys with it

Reply:
...as in, it prevents *all* movement._

Reply:
you mean, you can't get pushed around pulled or anything?